python/LICENSE.header

```diff
@@ -0,0 +1,13 @@
+Copyright 2023 Marcel Bollmann <marcel@bollmann.me>
```

Comment:
What is this file for?

Reply:
This is what gets placed at the top of new files. Although with more than one contributor, maybe we want to do that manually now.